### PR TITLE
disable all-day when clicking on all-day

### PR DESCRIPTION
**Pull request by georgehrke**
please review @nextcloud/calendar @eppfel

### CODE REVIEW

**Comment by georgehrke** [comment]
ref #72

File: css/app/eventdialog.css [hunk 1]
```diff
@@ -120,6 +120,10 @@
 	width: 60px;
 }
 
+.events .events--time[disabled] {
```
Comment (georgehrke) [inline css/app/eventdialog.css]: https://developer.mozilla.org/en-US/docs/Web/CSS/pointer-events

click doesn't work on disabled inputs. With `pointer-events:none` the clicks on the input do straight to the parent div

**Comment by georgehrke** [comment]
@eppfel please check again :)

**Comment by eppfel** [comment]
Still no text cursor, but with the at least with the mouse no second click is needed...

**Comment by eppfel** [comment]
Oh, but the time input is now in a new line, did this happen in master or is this regression?

**Comment by georgehrke** [comment]
this also happens in master

**Comment by georgehrke** [comment]
but this might be related to the span, I'll check

**Comment by georgehrke** [comment]
fixed that :)

**Comment by eppfel** [comment]
I approve once more 👍

**Pull request merged**